Tools/AP_Periph/Parameters.cpp

```diff
@@ -732,7 +732,7 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @User: Standard
     GSCALAR(imu_sample_rate, "INS_SAMPLE_RATE", 0),
 
-    // @Group: INS_
+    // @Group: INS
```

Comment:
Follows the same standard as: [ArduPlane/Parameters.cpp](https://github.com/ArduPilot/ardupilot/blob/be5c68d74db69dd2f3e5226ff576e1b9b8166c79/ArduPlane/Parameters.cpp#L910)